assets/src/components/map.tsx

```diff
@@ -251,49 +246,42 @@ export const autoCenter = (
   }
 }
 
-class RecenterControl extends Control {
-  private recenter: () => void
-  constructor(props: ControlOptions, recenter: () => void) {
-    super(props)
-    this.recenter = recenter
-  }
-
-  onAdd() {
-    const controlContainer = DomUtil.create(
-      "div",
-      "leaflet-control leaflet-bar m-vehicle-map__recenter-button"
-    )
-    controlContainer.onclick = (e) => {
-      e.stopPropagation()
-      e.preventDefault()
-      this.recenter()
-    }
-    controlContainer.innerHTML = `
-        <a
-          href="#"
-          title="Recenter Map"
-          role="button"
-          aria-label="Recenter Map"
+const RecenterControlButton = ({
+  position,
+  recenter,
+}: {
+  position: ControlPosition
+  recenter: () => void
+}): ReactElement => (
+  <Control position={position}>
```

Comment:
When the VPP is open on the search page, the recenter button attaches to the search map instead of the VPP. I think the map ref needs to be passed in some way to the control with the `container` prop

Reply:
After discussion, creating [⚙️ Improve map recenter control](https://app.asana.com/0/1152340551558956/1203142430154314/f) as a followup to improve this code and dropping the commit that did this particular refactor.